Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -304,11 +303,27 @@ - (void)disable:(BOOL)disabled
     }
 
     for (NSString *fieldID in fields) {
-        BOOL shouldDisable = (![fieldID isEqualToString:@"blank"] && ![self.formsManager.disabledFieldsIDs containsObject:fieldID]);
+        HYPFormField *field = [fields valueForKey:fieldID];
+        BOOL shouldChangeState = (![self.formsManager.disabledFieldsIDs containsObject:fieldID]);
```

Comment:
We no longer use blank fields in the layout so this should no longer be necessary